code/game/atoms_movable.dm

```diff
@@ -223,6 +223,9 @@
 				return
 		A.Bumped(src)
 
+/atom/movable/Crossed(atom/movable/O, oldloc)
+	. = ..(O)

```

Comment:
Um this is kinda necessary for tracking if someone already passed through a portal...